src/main/java/org/broadinstitute/hellbender/utils/help/GATKHelpDocWorkUnitHandler.java

```diff
@@ -55,6 +75,19 @@ protected void addCustomBindings(final DocWorkUnit currentWorkUnit) {
         if (picard.cmdline.CommandLineProgram.class.isAssignableFrom(toolClass)) {
             final CommandLineProgramProperties clpProperties = currentWorkUnit.getCommandLineProperties();
             currentWorkUnit.setProperty("picardsummary", clpProperties.summary());
+        } else if (MetricBase.class.isAssignableFrom(toolClass)) {
+            currentWorkUnit.setProperty(WORK_UNIT_SUMMARY_KEY, currentWorkUnit.getSummary());
+            final List<Map<String, String>> workUnitMetricsList = new ArrayList<>();
+            currentWorkUnit.setProperty(METRICS_MAP_ENTRY_KEY, workUnitMetricsList);
+            final com.sun.javadoc.FieldDoc[] fieldDocs = currentWorkUnit.getClassDoc().fields(false);
```

Comment:
Why fully qualify this instead of importing it?

Reply:
Not sure why this was like this, but this code has now been replaced with the Java 17 version since I rebase on that, so its gone anyway.